app/components/elements/FontStyles.js

```diff
@@ -205,9 +205,10 @@ export const TopNavLink = Styled(NavLink)`
 
 export const TextLink = Styled(Link)`
```

Comment:
Just a thought, but I think it would be more flexible to not bother with using styled-components for this element and sticking with the rebass `Link` element on it's own, and styling it via a theme variant.

Would allow this PR to be reduced to updating the `baseTheme.js` export to 
```js
export default {
  ...
  link: {
    fontFamily: fonts.default,
    textDecoration: 'none',
    color: colors.text.link,
  },
 ...
};
```

Then, in the storybook, just import `Link` directly from `rebass/styled-components` and it will automatically use the appropriate theme styles.

It would also allow us to leverage theme variants (say, if we wanted to add an `underlined` variant, or an `inverted` variant for links on colored backgrounds) if the need arises, with a couple simple changes to the base theme.

Reply:
To illustrate how I would add variants, I uploaded a `web-584-text-link-alt` branch that includes a couple of mock variants, and some rough storybook examples of how the various ways we could go about the Links differ (i.e. the `TextLink` component vs. using the `TextLink` styled component export from `FontStyles.js` vs. using the rebass `Link` directly with the theme variants.

Reply:
Thanks for the guidance -- I've gone ahead and made the suggested changes 👍